lib/hyrax/configuration.rb

```diff
@@ -425,10 +425,10 @@ def model_to_create
 
     private
 
-      # @param [Symbol] the symbol representing the model
+      # @param [Symbol, #to_s] concern_name - symbol representing the model
```

Comment:
Good catch…I changed my mind on the parameter name but forgot to change the documentation.